msteams-platform/concepts/deploy-and-publish/appsource/submit-your-app.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Submit your app to Partner Center
+description: Learn how to create a Partner Center account and submit your app for store validation.
+author: heath-hamilton
+ms.author: lajanuar
+ms.topic: how-to
+---
+# Submit your Microsoft Teams app to Partner Center
+
+After preparing your store submission, you can officially submit your app for review.
+
+## 1. Create a Partner Center account
+
+To publish your app to the Teams store and AppSource, you must [first set up a developer account](https://docs.microsoft.com/office/dev/store/open-a-developer-account).
+
+## 2. Submit your app
+
+Follow these [step-by-step store submission instructions](https://docs.microsoft.com/office/dev/store/add-in-submission-guide). When creating a new submission, specify you're submitting a Teams app.
+
+### App category mapping
+
+During submission, you're asked to categorize your app. The following table maps Teams store categories to the categories listed in Partner Center.
+
+| Teams categories       | Partner Center categories  |
+|:---------------------|:---------------|
+| Analytics and BI | Analytics, Data Visualization and BI |
+| Developer and IT | Developer Tools, IT Admin |
+| Education | Education |
+| Human resources | Human Resources and Recruiting |
+| Productivity | Content Management, Files and documents, Productivity, Training and Tutorials, and Utilities |
+| Project management | Communication, Project Management, Workflow, and Business Management |
+| Sales and support | Customer and Contact Management, Customer Support, Financial Management, Sales and Marketing |
+| Social and fun | Image and Video Galleries, Lifestyle, News and Weather, Social, Travel, and Navigation |
+
+### 3. Fix issues with your submission
+
+If your app fails submission, you receive a failure report so you know what to fix and resubmit. Microsoft also provides a white-glove service to help get your app listed.
```

Comment:
Do we have any info about how to sign up or register for the white-glove service?